### PR TITLE
Mcd encap prep cleanups 1

### PR DESCRIPTION
**Pull request by cgwalters**
Extracted from https://github.com/openshift/machine-config-operator/pull/935

With one other small commit on top. 

Would like to land this so I'm less in rebase hell later...

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -833,7 +829,7 @@ func (dn *Daemon) checkStateOnFirstRun() error {
 	// and if we still have a pendingConfig it means we've been killed by kube after 600s
 	// take a stab at that and re-run the drain+reboot routine
 	if state.pendingConfig != nil && bootID == dn.bootID {
-		dn.logSystem("killed by kube, retrying...")
+		dn.logSystem("drain interrupted, retrying")
```
Comment (runcom) [inline pkg/daemon/daemon.go]: thanks for this, I completely forgot to clean this up

**Comment by runcom** [comment]
/approve

I'm not sure we can land this even if it's just a cleanup and it's far better than before - I'll check that out and/or create a `master-4.3` branch otherwise where we can land these kind of PRs from now on

**Comment by cgwalters** [comment]
It's just feature freeze, not code freeze right?

**Comment by runcom** [comment]
> It's just feature freeze, not code freeze right?

right :thinking: this isn't changing anything then or adding anything

/lgtm

**Pull request merged**